Makefile

```diff
@@ -95,7 +96,7 @@ build:
 
 .PHONY: setup
 setup: build
-	#docker-compose scale elasticsearch=3
+	docker-compose up -d
```

Comment:
mmm ok but *start* is never called in the Makefile. If you want the *docker-compose up -d* inside the *start* I should change also *tests* and *test* the have inside **make setup** with **make start**. How does it sounds ?